packages/database/src/migrations/20201126075832-ChangeRawDataDownloadSurveyCodeFromCDtoDIA-modifies-data.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const reportIds = `'Raw_Data_CK_Surveys',
+                  'Raw_Data_DL_Surveys',
+                  'Raw_Data_VU_Surveys',
+                  'Raw_Data_TL_Surveys',
+                  'Raw_Data_KI_Surveys',
+                  'Raw_Data_TK_Surveys',
+                  'Raw_Data_SB_Surveys',
+                  'Raw_Data_VE_Surveys',
+                  'Raw_Data_TO_Surveys'`;
+
+exports.up = function(db) {
+  return db.runSql(`
+    update "dashboardReport" 
+    set "dataBuilderConfig" = jsonb_set("dataBuilderConfig", '{surveys,2,code}', '"DIA"') 
```

Comment:
Might have been preferable to have a fancier way of discovering index for each report but as this survey is listed in the same position for each report we can get away with supplying an index.